### PR TITLE
Update console to use latest operator

### PR DESCRIPTION
**Pull request by cesnietor**
Main changes Rename everything from Zone->Pool

Test Notes:
Please test a tenant is deployed correctly with Pools defined
Also add zone
Delete tenant

and make sure everything was renamed correctly (no typos)

### CODE REVIEW

**Review by harshavardhana — APPROVED** [review]

**Review by harshavardhana — DISMISSED** [review]

File: pkg/utils/parity.go [hunk 1]
```diff
@@ -185,9 +185,9 @@ func getTotalSizes(argPatterns []ellipses.ArgPattern) []uint64 {
 }
 
 // PossibleParityValues returns possible parities for input args,
-// parties are calculated in uniform manner for one zone or
-// multiple zones, ensuring that parities returned are common
-// and applicable across all zones.
+// parties are calculated in uniform manner for one pool or
+// multiple poolss, ensuring that parities returned are common
```
Comment (bexsoft) [inline pkg/utils/parity.go:189]: Please fix typo

**Review by bexsoft — CHANGES_REQUESTED** [review]
I couldn't be able to create a new tenant, the following error was thrown in log:

Screen Shot 2020-12-06 at 20 26 07

I'm using operator@3.0.29, Do we need an special version of operator?

**Comment by harshavardhana** [comment]
> I couldn't be able to create a new tenant, the following error was thrown in log:
> 
> Screen Shot 2020-12-06 at 20 26 07
> 
> I'm using operator@3.0.29, Do we need an special version of operator?

You need to use latest operator from master @bexsoft

File: pkg/utils/parity.go (hunk 1 above)
Comment (cesnietor) [reply pkg/utils/parity.go]: done

**Review by harshavardhana — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]
LGTM

**Review by bexsoft — APPROVED** [review]
LGTM

**Pull request merged**